docs/user/user/metrics/glean-event-listener.md

```diff
@@ -0,0 +1,67 @@
+# Glean Event Listener
```

Comment:
not a metrics API, this is a General API, so maybe it should be in `user/reference/general/` like `experiments-api.md`